src/components/screens/delegates/votingHeader.css

```
@@ -2,14 +2,154 @@
 @import './../../../app/mixins.css';
 
 .wrapper {
-  display: flex;
-  justify-content: space-between;
   width: 100%;
-  margin-bottom: 32px;
-  margin-top: 15px;
+  height: 112px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  box-sizing: border-box;
+  position: relative;
+  z-index: 2;
+
+  & .stickyContent {
+    position: relative;
+    height: 78px;
+    z-index: 9;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  & .bg {
+    position: fixed;
+    width: 100%;
+    height: 78px;
+    left: 0;
+    top: 73px;
+    z-index: 0;
+    border-bottom: solid 1px #e1e3eb;
```

For all colors we use variables like this.

```suggestion
    border-bottom: solid 1px var(--color-platinum);
```

---

Done